modules/jaggery-apps/app-store-web/extensions/assets/webapp/themes/store/js/logic/asset/overview/overview.js

```diff
@@ -17,6 +17,30 @@
  */
 
 $(function () {
+
+    //if user directly access the overview page, check the anonymous condition.
+    //if anonymous mode is not allowed and user is not authenticated, prompt the login page
+    var loggedUser = $("#hdnUsertId").val();
+    var allowAnonymous = $("#allowAnonymous").val();
+
+    //hide html body when loading
+    $('#wrap').css('visibility', 'hidden');
+
+    if (loggedUser == "" || loggedUser == null) {
```

Comment:
Removing checking this constraints as the body is already commented.
commit# https://github.com/wso2/carbon-appmgt/commit/917fab9c7323f9005453704fd6e2ff2d45dc3ec9